brd-ios/breadwallet/src/Redesign/Scenes/Swap/SwapMain/SwapMainViewController.swift

```diff
@@ -0,0 +1,146 @@
+//
+//  SwapMainViewController.swift
+//  breadwallet
+//
+//  Created by Kenan Mamedoff on 05/07/2022.
+//  Copyright © 2022 Fabriik Exchange, LLC. All rights reserved.
+//
+//  See the LICENSE file at the project root for license information.
+//
+
+import AVFoundation
+import UIKit
+
+class SwapMainViewController: BaseTableViewController<KYCCoordinator,
+                              SwapMainInteractor,
+                              SwapMainPresenter,
+                              SwapMainStore>,
+                              SwapMainResponseDisplays {
+    
+    typealias Models = SwapMainModels
+    
+    override var sceneLeftAlignedTitle: String? {
```

Comment:
what is this again? the "large titles" hack?

Reply:
large titles hack that acts like Tina behaved. i think i can replace this with large titles in the future. (the problem was, it needs to scroll and disappear instead of scrolling and being in middle of nav bar.). Added a comment.